curation_portal/migrations/0017_auto_20230726_0322.py

```diff
@@ -0,0 +1,83 @@
+# Generated by Django 2.2.24 on 2023-07-26 03:22
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("curation_portal", "0016_auto_20230614_0004"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="variant",
+            name="DP",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.IntegerField(blank=True, null=True),
+                blank=True,
+                null=True,
+                size=None,
+            ),
+        ),
+        migrations.AddField(
+            model_name="variant",
+            name="GQ",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.IntegerField(blank=True, null=True),
+                blank=True,
+                null=True,
+                size=None,
+            ),
+        ),
+        migrations.AddField(
+            model_name="variant",
+            name="GT",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.CharField(blank=True, max_length=1000),
+                blank=True,
+                null=True,
+                size=None,
+            ),
+        ),
+        migrations.AddField(
+            model_name="variant",
+            name="n_heterozygotes",
+            field=models.IntegerField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name="variant",
+            name="sample_ids",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.CharField(blank=True, max_length=1000),
+                blank=True,
+                null=True,
+                size=None,
+            ),
+        ),
+        migrations.AddField(
+            model_name="variantannotation",
+            name="appris",
+            field=models.CharField(blank=True, max_length=200, null=True),
+        ),
+        migrations.AddField(
+            model_name="variantannotation",
+            name="exon",
+            field=models.CharField(blank=True, max_length=200, null=True),
+        ),
+        migrations.AddField(
+            model_name="variantannotation",
+            name="hgvsc",
```

Comment:
Should confirm with @cassimons if 200 characters is sufficient for HGVS string fields. I doubt they would be longer than this, but just in case there could be an edge case where an insertion or deletion is really long.

Reply:
yea, I would be generous here as there can be some corner cases where this the hgvs fields could end up unreasonably large. We probably will not hit them, but no reason to barf if we do. I would just make is `1000` or something like that.